pkg/kubelet/cm/container_manager_stub.go

```diff
@@ -145,7 +145,7 @@ func (cm *containerManagerStub) ShouldResetExtendedResourceCapacity() bool {
 }
 
 func (cm *containerManagerStub) GetAllocateResourcesPodAdmitHandler() lifecycle.PodAdmitHandler {
-	return topologymanager.NewFakeManager()
+	return topologymanager.NewFakeManager(context.TODO())
```

Comment:
It would be nice to add a comment everywhere when context.TODO() is called, describing when and how this will be replaced by a valid context.

Reply:
Done.